applications/ConstitutiveLawsApplication/constitutive_laws_application.cpp

```diff
@@ -516,7 +518,6 @@ void KratosConstitutiveLawsApplication::Register()
     KRATOS_REGISTER_VARIABLE(ACCUMULATED_PLASTIC_STRAIN)
     KRATOS_REGISTER_VARIABLE(BACK_STRESS_VECTOR)
     KRATOS_REGISTER_VARIABLE(BACK_STRESS_TENSOR)
-    KRATOS_REGISTER_VARIABLE(HARDENING_MODULI_VECTOR)
```

Comment:
Why removing this?

Reply:
We originally added it for this CLs, but now we changed some of the parameters and this one no longer applies. I checked and it's not in use in Kratos, and tests run normally.

Reply:
Is it OK or should I leave it?